src/SonarScanner.MSBuild.Common/SonarProduct.cs

```diff
@@ -20,21 +20,20 @@
 
 using System;
 
-namespace SonarScanner.MSBuild.Common
+namespace SonarScanner.MSBuild.Common;
+
+public static class SonarProduct
 {
-    public static class SonarProduct
-    {
-        private static readonly Version SonarQube80 = new(8, 0, 0, 29455); // Build number of SQ 8.0
+    private static readonly Version SonarQube80 = new(8, 0, 0, 29455); // Build number of SQ 8.0
 
-        public static string GetSonarProductToLog(string host) =>
-            ContainsSonarCloud(host) ? "SonarCloud" : "SonarQube";
+    public static string GetSonarProductToLog(string host) =>
+        ContainsSonarCloud(host) ? "SonarCloud" : "SonarQube";
 
-        public static bool IsSonarCloud(string host, Version version) =>
-            version.Major == 8
-            && version.Minor == 0
-            && (version != SonarQube80 || ContainsSonarCloud(host));
+    public static bool IsSonarCloud(Version version) =>
+        version.Major == 8
+        && version.Minor == 0
+        && version != SonarQube80;
```

Comment:
Can we double-check with someone from the SC team that the version never changes? I would like to change the logic to check the exact Sonar Cloud version: https://sonarcloud.io/api/server/version.

This way, we will look specifically for the sonar cloud and avoid this indirect "is not SQ" check.

Reply:
Based on the fact that [production](https://sonarcloud.io/api/server/version) and [staging](https://sc-staging.io/api/server/version) are different, I am not sure we can do this.

I will ask a question to make sure anyways, since I like the suggestion.

Reply:
Unfortunately, the version changes. ([internal click](https://sonarsource.slack.com/archives/C04JP9PHHP1/p1720008284521179))

Reply:
Thanks for clarifying how it works!

Reply:
Would be awesome to have this as a comment in the code.